### PR TITLE
Change to package references

### PR DESCRIPTION
**Pull request by pascalberger**
Change to package references

### CODE REVIEW

File: src/Cake.Issues.PullRequests.Tfs.Tests/Cake.Issues.PullRequests.Tfs.Tests.csproj [hunk 1]
```diff
@@ -36,21 +35,23 @@
...
-    <Reference Include="Shouldly, Version=2.8.3.0, Culture=neutral, PublicKeyToken=6042cbcb05cbc941, processorArchitecture=MSIL">
-      <HintPath>..\packages\Shouldly.2.8.3\lib\net451\Shouldly.dll</HintPath>
-    </Reference>
+    <PackageReference Include="Cake.Core" Version="0.22.0" />
+    <PackageReference Include="Cake.Issues" Version="0.1.0" />
+    <PackageReference Include="Cake.Issues.Testing" Version="0.1.0" />
+    <PackageReference Include="Cake.Testing" Version="0.22.0" />
+    <PackageReference Include="Shouldly" Version="2.8.3" />
+    <PackageReference Include="StyleCop.Analyzers">
+      <Version>1.0.2</Version>
```
Comment (christianbumann) [inline src/Cake.Issues.PullRequests.Tfs.Tests/Cake.Issues.PullRequests.Tfs.Tests.csproj]: Move the version to line 43

File: src/Cake.Issues.PullRequests.Tfs/Cake.Issues.PullRequests.Tfs.csproj [hunk 2]
```diff
@@ -37,98 +37,40 @@
...
-      <HintPath>..\packages\Microsoft.VisualStudio.Services.Client.15.112.1\lib\net45\Microsoft.VisualStudio.Services.WebApi.dll</HintPath>
-    </Reference>
-    <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\packages\Newtonsoft.Json.10.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
-    </Reference>
+    <PackageReference Include="Cake.Core" Version="0.22.0" />
+    <PackageReference Include="Cake.Issues" Version="0.1.0" />
+    <PackageReference Include="Cake.Issues.PullRequests" Version="0.1.0" />
+    <PackageReference Include="Costura.Fody">
+      <Version>1.6.2</Version>
```
Comment (christianbumann) [inline src/Cake.Issues.PullRequests.Tfs/Cake.Issues.PullRequests.Tfs.csproj]: Move version to line 43. Same for other package references below.

File: src/Cake.Issues.PullRequests.Tfs.Tests/Cake.Issues.PullRequests.Tfs.Tests.csproj (hunk 1 above)
Comment (pascalberger) [reply src/Cake.Issues.PullRequests.Tfs.Tests/Cake.Issues.PullRequests.Tfs.Tests.csproj]: It's written like this by Visual Studio

File: src/Cake.Issues.PullRequests.Tfs/Cake.Issues.PullRequests.Tfs.csproj (hunk 2 above)
Comment (pascalberger) [reply src/Cake.Issues.PullRequests.Tfs/Cake.Issues.PullRequests.Tfs.csproj]: It's written like this by Visual Studio

**Pull request merged**